src/api/entities/Asset/types.ts

```diff
@@ -364,7 +364,7 @@ export enum TransferError {
 }
 
 export interface AssetWithGroup {
-  asset: FungibleAsset;
+  asset: FungibleAsset | NftCollection;
```

Comment:
```suggestion
  asset: Asset;
```
There is an alias which makes it look less intimidating in the docs.